cassandra/cluster.py

```diff
@@ -3679,6 +3712,9 @@ def _try_connect(self, host):
             shared_results = (peers_result, local_result)
             self._refresh_node_list_and_token_map(connection, preloaded_results=shared_results)
             self._refresh_schema(connection, preloaded_results=shared_results, schema_agreement_wait=-1)
+
+            if self._cluster.check_tablets_enabled():
+                self._refresh_tablets(connection)
```

Comment:
We do we have to call `_refresh_tablets` again? It is called a few lines above, so it seems redundant.

Reply:
Hm, now it is redundant, because we call `_refresh_tablets` to check if tablets are enabled, previously we called it after delay so this "second" call was actually first (initial) call

Reply:
removed that